test/test.js

```diff
@@ -86,9 +86,12 @@ function testInput(input) {
       });
 
       it('should generate valid Swagger 2.0 document', function() {
-        SwaggerTools.specs.v2.validate(converted, function(validationErrors) {
-          expect(validationErrors).to.be.undefined;
-        });
+        function validateCallback(validationErrors, validationResults) {
+          var errors = [].concat(validationErrors || [])
```

Comment:
I made this to make test more verbose.
Chai is log difference between received and expected values. 
So if it fails it would actually log errors.
In other case you just know that some random error happened, without any specific info.
It especially true for Travis, validation errors should be in log file.